src/wp-includes/general-template.php

```diff
@@ -1436,7 +1436,7 @@ function wp_title( $sep = '&raquo;', $display = true, $seplocation = '' ) {
 	 *
 	 * @param string[] $title_array Array of parts of the page title.
 	 */
-	$title_array = apply_filters( 'wp_title_parts', explode( $t_sep, $title ) );
+	$title_array = apply_filters( 'wp_title_parts', explode( $t_sep, $title ?? '' ) );
```

Comment:
This seems better:

```suggestion
	$title_array = apply_filters( 'wp_title_parts', ! empty( $title ) ? explode( $t_sep, $title ) : '' );
```

No need to try exploding an empty string.